contracts/src/account.rs

```diff
@@ -0,0 +1,23 @@
+use near_sdk::{ext_contract, near, AccountId};
+
+// TODO: make this contract an extension of NFT, since every Account
+// is a unique item. This will ease the integration of Defuse
+// with already existing tooling around NFTs
+#[ext_contract(ext_account_contract)]
+pub trait AccountContract {
+    /// Create an account with given defivation path for given owner
+    fn create_account(&mut self, owner: AccountId, derivation_path: String);
+    /// Change an owner of account with given derivation path.
+    fn change_owner(&mut self, from: &AccountId, to: AccountId, derivation_path: String);
+    /// Return all [`Account`]s owned by given `owner`
+    fn get_accounts(&self, owner: &AccountId) -> Vec<(String, Account)>;
+
+    /// Return MPC contract for this
+    fn mpc_contract(&self) -> &AccountId;
+}
+
+#[derive(Debug, Clone, Default, Eq, PartialEq)]
+#[near(serializers=[borsh, json])]
+pub struct Account {
```

Comment:
I would propose promoting the module to directory and placing types to the `types.rs` and traits describe in mod.rs file:
```
-- account
    |_ mod.rs
    |_ types.rs 
```